opentable/oc
# Exit 1 in case of CLI error

Fixes #174 

Now that all the cli's facades have an optional callback (not used by the cli) - it is possible, in case it is not provided, to wrap it with a utility that exits with a `process.exit(1)` when a callback is fired with an error.

To test locally with unix/mac: 
* checkout the branch locally
* run a command that should fail like `node oc-cli.js registry add 23456`
* run `echo $?` to see previous command's exit code
